pkg/apis/pytorch/v1/doc.go

```diff
@@ -12,7 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// +k8s:deepcopy-gen=package,register
```

Comment:
We don't need `deepcopy-gen` to generate file anymore since we have `controller-gen` from kubebuilder. They do the same work